changelog.md

```diff
@@ -1,6 +1,6 @@
 # Changelog for [`process` package](http://hackage.haskell.org/package/process)
 
-## Unreleased changes
+## 1.6.5.2 *September 2019*
```

Comment:
Version bump should be to 1.6.6. Also, please include a `@since` comment on the new Haddocks.

Reply:
Done